python/mkdocs/docs/configuration.md

```diff
@@ -85,3 +85,15 @@ catalog:
   default:
     type: glue
 ```
+
+## DynamoDB Catalog
+
+If you want to use AWS DynamoDB as the catalog, you can use the last two ways to configure the pyiceberg and refer
+[How to configure AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+to set your AWS account credentials locally.
+
+```yaml
+catalog:
+  default:
+    type: dynamodb
```

Comment:
I think table name is required for dynamoDB catalog

Reply:
Added.